Library/Homebrew/os/linux/elf.rb

```diff
@@ -87,15 +87,18 @@ def with_interpreter?
   end
 
   def dynamic_elf?
-    return @dynamic_elf if defined? @dynamic_elf
+    return @dynamic_elf if defined?(@dynamic_elf)
```

Comment:
I personally prefer without parentheses when they're not required stylistically, so I would leave this line as is. I don't know whether Homebrew has an official style policy on this point.

Reply:
This change is unrelated to the PR so should be left as-is.